cmd/util/util.go

```diff
@@ -1,4 +1,4 @@
-// Package util provides common utilities for spf13/cobra CLI utilities
+// Package util provides sqlcommon utilities for spf13/cobra CLI utilities
```

Comment:
This doesn't need to change. It's referring to the literal word "common" not the package `common`.

Reply:
fixed